openverse_catalog/dags/common/github.py

```diff
@@ -4,6 +4,8 @@
 class GitHubAPI:
     def __init__(self, pat: str):
         """
+        Initialize.
```

Comment:
This is _really_ tedious to have to include :disappointed: I don't understand why pydocstyle requires a summary for `__init__`.

Maybe we should consider ignoring D402, which I believe is what forces this?